packages/quantic/force-app/main/default/lwc/quanticSearchBoxSuggestionsList/quanticSearchBoxSuggestionsList.css

```diff
@@ -0,0 +1,3 @@
+.quantic-suggestions-list {
+  margin-top: 0;
```

Comment:
why do we want this here is it to reset default margin or something?

Reply:
There was a margin in the Quantic Query suggest look which created a gap between the suggestions and the bottom of the searchbox. It looked weird so after discussion with UX we removed it. It's connected in Atomic as well so I dunno why we had it.